my-app/src/Api.js

```diff
@@ -9,17 +9,29 @@ const continueReading = (articleID) => {
   return `<p class="continue-reading"><a href="${articleID}">Continue reading...</a></p>`;
 };
 
-// TODO: replace modern expressions with 19th century equivalents.
+// replace modern words/expressions with 19th century equivalents.
+const replaceWords = (articleBody) => {
+  const mapObj = {
+    you: "thou",
+    know: "knowest"
+  };
+  let articleOldStyle = articleBody.replaceAll(/you|know/g, (matched) => {
+    return mapObj[matched]
+  });
+
+  return articleOldStyle
+}
```

Comment:
I would change this function to something like this:

```javascript
const replaceWords = (text) => {
  const rules = [
     [/\byou\b/g, "thou"],
     [/(from which|from where)/g, "whence"]
  ]
  rules.forEach(rule => {
    const regex = rule[0]
    const replacement = rule[1]
    text = text.replace(regex, replacement)
  })
  return text
}
```

The function contains a list of rules (`rules`) where each element is an array consisting of two elements, the expression to match and its substitution. This way if you want to add more rules, you just extend `rules`.
`\b` defines a word boundary in a regular expression which removes the need to add spaces and avoids transforming words like `youth` to `thouth`.